package/pfazuredb/tools/postgrespg.py

```diff
@@ -20,7 +20,7 @@ def vectorsearch(
     import json
 
     # establish connection
-    pgconnection = psycopg2.connect(connection.configs["conn_string"])
+    pgconnection = psycopg2.connect(connection.secrets["AZURE_COSMOSDB_POSTGRES_CONN_STRING"])
```

Comment:
this needs to be pgconnection = psycopg2.connect(connection.secrets["AZURE_POSTGRES_CONN_STRING"])